### DISCUSSION TITLE
Some nightlies are no longer updated

### DISCUSSION THREAD

**Discussion by Kabouik** [KRTirtho/spotube#1165] [Q&A]
![ss20240127-164404](https://github.com/KRTirtho/spotube/assets/7107523/53a6b67d-fd35-482b-9d4a-60b8c4630746)

Some of the nightlies have not been updated since late 2023, and also I believe some of them are [actually not built for the architecture they are made for](https://github.com/KRTirtho/spotube/issues/715#issuecomment-1887532478). Would it please be possible to fix that and add them back to the pipeline? This would be amazing.

**Comment by KRTirtho** [comment]
The aarch64 (arm) ones were experimentally built. But unfortunately it doesn't work well. But no worries I think I can build them using docker on apple m1

GitHub actions don't have any arm runner. So that's the problem.

**Reply by KRTirtho** [reply]
The tar.xz is the confusing one. Maybe it's not getting pushed into nightly for some reason
